tests/Commands/UserModelGeneratorTest.php

```diff
@@ -18,10 +19,17 @@ protected function setUp(): void
     {
         parent::setUp();
 
-        CITestStreamFilter::$buffer = '';
+        // @phpstan-ignore-next-line
+        if (version_compare(CodeIgniter::CI_VERSION, '4.3.0', '>=')) {
```

Comment:
Why do you need this if statement? Why not go with 4.3 and change related lines?

Reply:
4.3.0 has many breaking changes. I think dropping 4.2 now is too early.

Reply:
Okay. I think that's the reason for your PR testing against lowest dependencies, right?

Reply:
Yes!